noxfile.py

```diff
@@ -80,7 +80,7 @@ def on_master_no_changes(session):
 
 @nox.session(python=PYTHON_ALL_VERSIONS)
 def tests(session):
-    session.install("--upgrade", "pip")
+    session.run("python", "-m", "pip", "install", "--upgrade", "pip")
```

Comment:
Should we file an issue on the nox project to switch to this?

Reply:
Hmm maybe.  It only is a problem for windows and `pip`.

I can file an issue.

Reply:
Looks like nox is about to merge a PR that does this very thing. :)